docs/styling.md

```diff
@@ -23,8 +23,8 @@ Our current styling architecture is still quite nascent and is very likely to un
 | ---------------------------------- | ------------ | --------------------- | ------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `--media-control-background`       | `background` | `rgba(20,20,30, 0.7)` | background color of the component                                                                 | Applies to other components as well ([See notes below \*\*](#notes))                                                                                                 |
 | `--media-control-hover-background` | `background` | `rgba(50,50,70, 0.7)` | background color of the button when hovered                                                       | Applied by the `:hover` [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:hover) Applies to other components as well ([See notes below \*\*](#notes)) |
```

Comment:
should add `--media-control-padding` and `--media-control-height` to the docs here